pxd_fastpath.c

```diff
@@ -780,7 +865,7 @@ void fastpath_queue_work(struct kthread_work* work, bool completion)
 		struct pxfpcontext_per_node *c = &pxfpctxt[node];
 		if (c->valid) {
 			cpuid = balanceIO(c, cpuid, completion);
-			worker = c->fpworker[cpuid & MAX_PXFP_WORKERS_PER_NODE_MASK];
+			worker = c->fpworker[cpuid % MAX_PXFP_WORKERS_PER_NODE];
```

Comment:
mod is an expensive call relatively compared to bit ops. dont see a way out if not a power of 2. But maybe there is a kernel macro that improves this is MAX_FP_WORKERS_PER_NODE turns out to be a power of 2.